src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,54 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+    if (typeof(text) != "string"){
+        throw "Please enter a valid string";
+    }
+    else{
```

Comment:
You don't need this `else` statement since your program would exit if the `if` condition is matched.

Reply:
Noted sir, I will apply this correction.